### ISSUE TITLE
Not Wokring In IOS  and samsung device

### ISSUE THREAD

**Issue by neha-logicalloop** [joaocarmo/vcard-creator#26]
In IOS it shows that contact was created but actually not created in Contacts  List
Samsung shows file not supported

**Comment by joaocarmo** [comment]
@neha-logicalloop can you give a bit more context? How are you generating the vCard? Are you using the [iCalendar format](https://github.com/joaocarmo/vcard-creator#icalendar-format) for Apple devices?

**Comment by neha-logicalloop** [comment]
> @neha-logicalloop can you give a bit more context? How are you generating the vCard? Are you using the [iCalendar format](https://github.com/joaocarmo/vcard-creator#icalendar-format) for Apple devices?

myVCard
            // Add personal data
            .addName(lastname, firstname, additional, prefix, suffix)
            // Add work data
            // .addCompany('Siesqo')
            // .addJobtitle('Web Developer')
            .addEmail(`${response.data.data.email}`)
            .addPhoneNumber(mobNum, "PREF;WORK")
            .addPhoto(response.data.data.profile_pic, "JPEG")
            .addPhoneNumber(mobNum1, "WORK");
          // .addURL('http://www.jeroendesloovere.be')

          var file = new Blob([myVCard.toString()], {
            type: "text/vcard;charset=utf-8",
          });
          var csvURL = window.URL.createObjectURL(file);
          let tempLink = document.createElement("a");
          tempLink.href = csvURL;
          tempLink.setAttribute("download", `${firstname}.vcf`);
          
          
          
          yes I am using this it's working cool in windows and android but Not in IOS and Samsung devices

**Comment by joaocarmo** [comment]
I'm not sure what's the problem with the Samsung devices, but for iOS you should use the [iCalendar format](https://github.com/joaocarmo/vcard-creator#icalendar-format). You can try something like the following.

```js
// Create vCard instance...

const isAppleMobile = ['iPad', 'iPhone', 'iPod'].includes(navigator.userAgentData.platform)

myVCard.setFormat(isAppleMobile ? 'vcalendar' : 'vcard')

// Create blob...
```

I'll investigate the other issue.

**Comment by CalumY** [comment]
FYI, it appears that the [iCalendar format](https://github.com/joaocarmo/vcard-creator#icalendar-format) is no longer necessary when adding a contact on an iPhone as the `.vcf` file extension is now supported.

**Comment by joaocarmo** [comment]
Resolved in #53.
